blocks/spinning-cards/spinning-cards.css

```diff
@@ -0,0 +1,76 @@
+.spinning-cards-grid  {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 30px;
+  margin: 0;
+  padding: 0;
+}
+
+.spinning-cards-item {
+  width: auto;
+  height: 250px;
+  perspective: 1000px;
+  float:left;
```

Comment:
this float declaration doesn't have any effect. you can remove it

Reply:
removed